cost/kubecost/sizing/README.md

```diff
@@ -0,0 +1,42 @@
+# Kubecost Request Sizing Recommendations Policy Template
+
+## What it does
+
+The Kubecost Request Sizing Recommendations Policy Template is used to get recommendations for resizing instances controlled with Kubecost
```

Comment:
Can we change this to ?
Kubecost request sizing recommendations policy template is used to resize instances controlled by Kubecost.